src/components/NavBar.jsx

```diff
@@ -1,27 +1,45 @@
 import { NavLink, Link } from 'react-router-dom';
+import { useState } from 'react';
 import logo from '../images/planet_1.png';
 
-const NavBar = () => (
-  <header>
-    <nav>
-      <Link to="/">
-        <img className="logo" src={logo} alt="" />
-        <h1>Space traveler&apos; Hub</h1>
-      </Link>
-      <ul>
-        <NavLink className="nav-items" to="/">
-          Rockets
-        </NavLink>
-        <NavLink className="nav-items" to="/missions">
-          Missions
-        </NavLink>
-        <span>|</span>
-        <NavLink className="nav-items" to="/profile">
-          My Profile
-        </NavLink>
-      </ul>
-    </nav>
-  </header>
-);
+const NavBar = () => {
+  const [active, setActive] = useState('Rockets');
+
+  return (
+    <header>
+      <nav>
+        <Link to="/">
+          <img className="logo" src={logo} alt="" />
```

Comment:
[optional] As a good practice of HTML & CSS leaving the `alt` attribute empty is not recommended. Thus, kindly provide a descriptive alternative text for the image to enhance accessibility.